packages/compose/README.md

```diff
@@ -90,7 +90,7 @@ _Parameters_
 
 _Returns_
 
--   `WPComponent`: Component class with generated display name assigned.
+-   `null`: Component class with generated display name assigned.
```

Comment:
@nosolosw - it looks like the following doesn't work:
`{import('@wordpress/element').WPComponent}`

Reply:
No, because it isn't JSDoc syntax. See my [comment](https://github.com/WordPress/gutenberg/pull/18025/files/1da5770513c08449305fd19d74d5e054f082c952#r336880417) above.